SWLOR.Game.Server/Feature/SpawnDefinition/DantooineResourceSpawnDefinition.cs

```diff
@@ -0,0 +1,213 @@
+﻿using System.Collections.Generic;
+using SWLOR.Game.Server.Core.NWScript.Enum;
+using SWLOR.Game.Server.Service.SpawnService;
+
+namespace SWLOR.Game.Server.Feature.SpawnDefinition
+{
+    public class DantooineResourceSpawnDefinition : ISpawnListDefinition
+    {
+        private readonly SpawnTableBuilder _builder = new();
+
+        public Dictionary<string, SpawnTable> BuildSpawnTables()
+        {
+            DanLake();
+            DanWildPlains();
+            DantooineWareHouse();
+            DanTribeVillage();
+            DantooineCrystalFields();
+            DantooineJantaCaves();
+            DanForsakenJungles();
+            DanDestroy();
+
+            return _builder.Build();
+        }
+
+        private void DanLake()
+        {
+            _builder.Create("DANTOOINE_DAN_LAKE_RESOURCES")
```

Comment:
The naming is weird here. Is "Dan Lake" an actual place or an abbreviation? If it's the latter, please remove it as it's redundant. Also update referencing NPCs when you do this.